content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -22,21 +22,131 @@ external components communicate with one another.
 The Kubernetes API lets you query and manipulate the state of API objects in Kubernetes
 (for example: Pods, Namespaces, ConfigMaps, and Events).
 
-Most operations can be performed through the
-[kubectl](/docs/reference/kubectl/) command-line interface or other
-command-line tools, such as
-[kubeadm](/docs/reference/setup-tools/kubeadm/), which in turn use the
-API. However, you can also access the API directly using REST calls.
+Most operations can be performed through the [kubectl](/docs/reference/kubectl/)
+command-line interface or other command-line tools, such as
+[kubeadm](/docs/reference/setup-tools/kubeadm/), which in turn use the API.
```

Comment:
Would it be useful to show any of the kubectl operations? api-resources maybe?

Reply:
It feels a bit weird to directly talk about api-resources in a section that introduces both discovery and OpenAPI. I added a note around api-resources in the discovery section.

Reply:
OK. On my first pass reading this I was wondering why kubectl was being discussed which prompted my question.  If we don't want to get into kubectl here, maybe don't bring it up?

Reply:
Giving examples of clients that use the API without going into the specifics. I removed the api-resources part, but I think it's still useful to link to clients like kubectl, kubeadm, etc?